components/emoji_picker/modules/emoji_selector.vue

```diff
@@ -65,15 +69,20 @@
           class="d-emoji-picker__tab "
         >
           <button
-            v-for="emoji in filteredEmojis"
+            v-for="(emoji, index) in filteredEmojis"
             :key="emoji.shortname"
+            :ref="el => { if (el) setFilteredRef(el, index) }"
             type="button"
             :aria-label="emoji.name"
+            :class="{
+              'hover-emoji': (index === 0 && hoverFirstEmoji),
+            }"
             @click="$emit('selected-emoji', emoji)"
             @focusin="$emit('highlighted-emoji', emoji)"
             @focusout="$emit('highlighted-emoji', null)"
-            @mouseover="$emit('highlighted-emoji', emoji)"
-            @mouseleave="$emit('highlighted-emoji', null)"
+            @mouseover="hoverEmoji(emoji)"
+            @mouseleave="hoverEmoji(null)"
+            @keydown="event => handleKeyDownFilteredEmojis(event, index, emoji)"
```

Comment:
the blue focus ring should not display on click, only when using keyboard for navigation (focus-visible style behaviour)

Reply:
Since the expected behaviour is to close the emojipicker on emoji selection I dont see this necessary.

Reply:
This isn't always the case since it's pretty common to keep the emoji picker open when shift is held and an emoji is clicked (test on slack for example)

Reply:
hmm that seems to be another accessibility thing to handle because that behaviour will probably not be possible.
Because on `click` you emit a `close` event but you need to catch the `shift` to not emit it or ...

I dont know how the user can replicate that case in our emoji picker

Reply:
I'm not sure we need to handle anything on our side, on the product side they could capture a shift key press event in their wrapper component and then decide whether they want to close it or not when the "selected-emoji" event triggers

Reply:
Okay...
This is because I added this css style

` &:focus{
        box-shadow: var(--bs-focus-ring);
      }`

On the button in `.d-emoji-picker{` class... this is necessary to have because when you click on a tab this focus the first emoji of the tab, then you can navigate it with keyboard.

Your request to remove it seems like to be a side effect and I need to investigate how I can replace it.

Reply:
can you just change `&:focus` to `&focus-visible`?